docs/source/notebooks/clv/bg-nbd.ipynb

```diff
@@ -28,8 +28,7 @@
    "outputs": [],
    "source": [
     "import arviz as az\n",
-    "import matplotlib.pyplot as plt\n",
-    "import aesara.tensor as at"
```

Comment:
Aren’t you missing an import from pytensor here?

Reply:
No, they were not being used